contrib/TestHarness2/test_harness/joshua.py

```diff
@@ -50,6 +50,8 @@ def _print_summary(summary: SummaryTree, commands: Set[str]):
         file_name = summary.attributes['TestFile']
         role = 'test' if test_harness.run.is_no_sim(Path(file_name)) else 'simulation'
         cmd += ['-r', role, '-f', file_name]
+        if "Restart-2" in file_name:
```

Comment:
This is not reliable. E.g., `restarting/from_5.2.0_until_6.3.0/ClientTransactionProfilingCorrectness-2.txt`.

Reply:
updated. thanks.